lib/genre.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './item'
+
+class Genre < Item
+  attr_accessor :name
+
+  def initialize(name)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
```

Comment:
According to the UMl diagram, genre is not a child of item but only has a `one-to-many` relationship with it. There is no need to make genre inherit from item and call the `super` method. Please change this🙏